vector/src/main/java/im/vector/adapters/ContactAdapter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 New Vector Ltd
+ * Copyright 2018 Dinsic
```

Comment:
Should be DINSIC, no?
Does it mean this file could be moved to a tchap java package?

Reply:
Yes it will be moved soon